_inc/lib/core-api/class-wpcom-rest-field-controller.php

```diff
@@ -161,55 +165,56 @@ private function prepare_for_response( $value, $request ) {
 	/**
 	 * Permission Check for the field's getter. Must be implemented in the inheriting class.
 	 *
-	 * @param mixed $object_data Whatever the endpoint would return for its response.
+	 * @param mixed           $object_data Whatever the endpoint would return for its response.
 	 * @param WP_REST_Request $request
 	 * @return true|WP_Error
 	 */
 	public function get_permission_check( $object_data, $request ) {
 		/* translators: %s: get_permission_check() */
-                _doing_it_wrong( 'WPCOM_REST_API_V2_Field_Controller::get_permission_check', sprintf( __( "Method '%s' must be overridden.", 'jetpack' ), __METHOD__ ), 'Jetpack 6.8' );
+				_doing_it_wrong( 'WPCOM_REST_API_V2_Field_Controller::get_permission_check', sprintf( __( "Method '%s' must be overridden.", 'jetpack' ), __METHOD__ ), 'Jetpack 6.8' );
```

Comment:
is this linting ok?

Reply:
no - looks like it got confused about some spaces. I'll fix.

Reply:
Thanks for noticing! Fixed in 6a22f1148.